package.json

```diff
@@ -4,6 +4,9 @@
   "description": "A Social platform for the creative at heart",
   "main": "index.js",
   "scripts": {
+    "start": "npm run-script build && node ./build/index.js",
```

Comment:
Nice :+1: .

One quick one though, I'll advice you to not run the build command in the `start` script. As the app grows bigger, the build will take longer to build and that will prevent heroku from binding to port and the app will crash.

You can use any of heroku custom build command. See here https://devcenter.heroku.com/articles/nodejs-support#customizing-the-build-process

I personally prefer running the build in a `postinstall` script.

Reply:
And why `run-script` instead of `run`?

Reply:
> Nice 👍 .
> 
> One quick one though, I'll advice you to not run the build command in the `start` script. As the app grows bigger, the build will take longer to build and that will prevent heroku from binding to port and the app will crash.
> 
> You can use any of heroku custom build command. See here https://devcenter.heroku.com/articles/nodejs-support#customizing-the-build-process
> 
> I personally prefer running the build in a `postinstall` script.

Oh, yes. This is fantastic, Thanks. Will effect changes immediately.

> And why `run-script` instead of `run`?

No particular reason. While testing the build script manually. I accidentally typed in `npm build` and I was auto corrected by npm with a message  "Did you mean to `npm run-script build`". So I continued to use it. But I realize it does the samething `npm run`. I will change it. Thanks